app/models/fuel/post.rb

```diff
@@ -20,6 +20,10 @@ class Post < ActiveRecord::Base
     scope :recent_published_posts, -> { where(published: true).order("created_at DESC") }
     scope :recent, -> { order("created_at DESC") }
 
+    def s3_credentials
+      {:bucket => Fuel.configuration.aws_bucket, :access_key_id => Fuel.configuration.aws_access_key, :secret_access_key => Fuel.configuration.aws_secret_access_key}
```

Comment:
Line is too long. [165/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.